overlord/snapshotstate/snapshotstate.go

```diff
@@ -197,6 +202,30 @@ func Restore(st *state.State, setID uint64, snapNames []string, users []string)
 
 	ts = state.NewTaskSet()
 
+	for i, snapID := range snapIDs {
+		if snapID == "" {
+			// snapshotted snap was sideloaded, never mind
```

Comment:
`// The snap referring to the snapshot was locally installed without assertions`